static/data/questions.json

```diff
@@ -64,20 +72,23 @@
         "options": ["SMS", "Email"],
         "value": "Preferred Contact",
         "helper": "What's the best way to reach you?",
+        "validator": "string",
         "followUpValue": "NONE"
       },
       {
         "key": "referer_contact",
         "type": "text",
         "value": "Your Contact",
         "helper": "Please provide your primary contact information.",
+        "validator": "contact",
```

Comment:
Let's generalize `contact` to a list validator, where the question has a list of appropriate values.